gem/styles/gem-springster/layout/_l-header.scss

```diff
@@ -3,7 +3,7 @@ html, body {
 }
 .header {
   overflow: auto;
-  padding: 10px 10px 5px;
+  padding: 10px 10px 0 5px;
```

Comment:
@projectmushroom thanks. I've increased this size on smart style - Nokia phones a a white line break there unless bottom padding is 0 or 10px. 

I'm really not sure why.